AK/StringUtils.cpp

```diff
@@ -92,7 +92,7 @@ bool matches(StringView str, StringView mask, CaseSensitivity case_sensitivity,
 }
 
 template<typename T>
-Optional<T> convert_to_int(StringView str, TrimWhitespace trim_whitespace)
+Optional<T> convert_to_int(StringView str, TrimWhitespace trim_whitespace, NumericParseMode parse_mode)
```

Comment:
couldn't the caller do this and pass in a smaller stringview then? seems a bit katamari-damacy-esque to add it here

Reply:
Yeah, I agree. I went back and forth if putting this inside AK was the right choice before submitting the first version.

I have now added a new function inside LibWeb that handles this in the caller instead.